src/com/company/ppj14/Task5.java

```diff
@@ -0,0 +1,55 @@
+package com.company.ppj14;
+
+public class Task5 {
+    public static void main(String[] args) {
+        int rows = 10;
+        int columns = rows;
+        int[][] array = new int[rows][columns];
```

Comment:
Let's implement the solution in other way.
We don't need the array, we just print values from the methods calculations.

For now implement just the 1st step - method  `private static void printCol(int row, int col, int size)`
which will use recursion and it prints colums only for one specific row.
Examples:
Call this method from `main()` with parameters  
- `printCol(5, 1, 10);`  it should print
"1234554321",
- `printCol(4, 1, 10);`  it should print
"1234444321", 
- `printCol(3, 1, 10);` it should print
"1233333321"
...